### ISSUE TITLE
Chat renderer image sharpening

### ISSUE THREAD

**Issue by swmarc** [lay295/TwitchDownloader#482]
### Checklist

- [X] I have checked the [issue page](https://github.com/lay295/TwitchDownloader/issues) for duplicates
- [X] I am running the latest version ([download here](https://github.com/lay295/TwitchDownloader/releases/latest))

### Write your feature request here

By using FFmpeg's video filter `smartblur`, which is also able to sharp images, text becomes way less blurry by
replacing the default FFmpeg input arguments from

`-framerate {fps} -f rawvideo -analyzeduration {max_int} -probesize {max_int} -pix_fmt bgra -video_size {width}x{height} -i -`
to
`-framerate {fps} -f rawvideo -analyzeduration {max_int} -probesize {max_int} -pix_fmt bgra -video_size {width}x{height} -filter_complex "smartblur=lr=1:ls=-1.0" -i -`

Probably this could be an tickable option in the UI and maybe even the default.
Encoding time is a bit higher and the resulting filesize is roughly 50% larger on my tests, but it's worth it.

**Comment by goldbattle** [comment]
Without (SPEED: 3.51x):
![image](https://user-images.githubusercontent.com/2222562/218239961-99b0f228-0be2-43c2-95e4-4701d5b91f5b.png)




With (SPEED: 3.21x):
![image](https://user-images.githubusercontent.com/2222562/218239864-926d1427-4cad-42f4-98a8-865d695510e3.png)

---

Without:
![image](https://user-images.githubusercontent.com/2222562/218240228-aa27ef5e-a18c-49b3-9e56-897304f44db9.png)

With:
![image](https://user-images.githubusercontent.com/2222562/218240288-230d9353-a0da-4fd0-bd24-5e4ab72e57cb.png)

---

Documentation:
https://ffmpeg.org/ffmpeg-filters.html#smartblur-1

```
luma_radius, lr
Set the luma radius. The option value must be a float number in the range [0.1,5.0] that specifies the variance of the gaussian filter used to blur the image (slower if larger). Default value is 1.0.

luma_strength, ls
Set the luma strength. The option value must be a float number in the range [-1.0,1.0] that configures the blurring. A value included in [0.0,1.0] will blur the image whereas a value included in [-1.0,0.0] will sharpen the image. Default value is 1.0.
```

Seems pretty nice. I wouldn't be against setting it as default, but would want others feedback first. Seems to make the images much more defined at lower resolutions, but does introduce some coloring artifacts around the text.

**Comment by ScrubN** [comment]
> Without (SPEED: 3.51x):
> 
> With (SPEED: 3.21x):
> 
> Documentation: https://ffmpeg.org/ffmpeg-filters.html#smartblur-1
> 
> ```
> luma_radius, lr
> Set the luma radius. The option value must be a float number in the range [0.1,5.0] that specifies the variance of the gaussian filter used to blur the image (slower if larger). Default value is 1.0.
> 
> luma_strength, ls
> Set the luma strength. The option value must be a float number in the range [-1.0,1.0] that configures the blurring. A value included in [0.0,1.0] will blur the image whereas a value included in [-1.0,0.0] will sharpen the image. Default value is 1.0.
> ```
> 
> Seems pretty nice. I wouldn't be against setting it as default, but would want others feedback first. Seems to make the images much more defined at lower resolutions, but does introduce some coloring artifacts around the text.

You first example looks awesome with the sharpening, though the second example I would personally turn it off. Maybe we should make it default enabled for GUI but default disabled for CLI since the default resolution is larger in the GUI.
